tests/unit_tests/gui/test_gui_load.py

```diff
@@ -185,6 +193,75 @@ def test_that_the_ui_show_warnings_when_there_are_no_observations(qapp, tmp_path
         assert gui.windowTitle() == "ERT - config.ert"
 
 
+@pytest.mark.usefixtures("copy_poly_case")
+def test_that_the_ui_show_warnings_when_parameters_are_missing(qapp, tmp_path):
+
+    with open("poly.ert", "r", encoding="utf-8") as fin:
+        with open("poly-no-gen-kw.ert", "w", encoding="utf-8") as fout:
+            for line in fin:
+                if "GEN_KW" not in line:
+                    fout.write(line)
+
+    args = Mock()
+
+    args.config = "poly-no-gen-kw.ert"
+    with add_gui_log_handler() as log_handler:
+        gui, _ = ert.gui.main._start_initial_gui_window(args, log_handler)
+        combo_box = gui.findChild(QComboBox, name="Simulation_mode")
+        assert combo_box.count() == 5
+
+        for i in range(2):
+            assert combo_box.model().item(i).isEnabled()
+        for i in range(2, 5):
+            assert not combo_box.model().item(i).isEnabled()
+
+        assert gui.windowTitle() == "ERT - poly-no-gen-kw.ert"
+
+
+@pytest.mark.parametrize(
+    "mode",
+    [
+        pytest.param(ENSEMBLE_SMOOTHER_MODE),
+        pytest.param(ITERATIVE_ENSEMBLE_SMOOTHER_MODE),
+        pytest.param(ES_MDA_MODE),
+    ],
+)
+@pytest.mark.usefixtures("copy_poly_case")
+def test_that_the_cli_raises_exceptions_when_parameters_are_missing(mode):
```

Comment:
Any hint on where this should reside?

Reply:
Don't have strong opinions on this, but seems fine to me to put it into `test_gui_load.py` as you have done here.

Reply:
`tests/unit_tests/cli/test_integrations_cli.py` seems a good fit.